gilda/api.py

```diff
@@ -122,5 +161,78 @@ def make_grounder(
         A Grounder instance, initialized with either the default terms
         loaded from the resource file or a custom set of terms
         if the terms argument was specified.
+
+    Examples
+    --------
+    While the NCBITaxon namespace is often used as the gold standard for species,
+    there are several complementary other species- and taxonomy-centric
+    namespaces, such as the `Integrated Taxonomic Information System
+    <https://bioregistry.io/registry/itis>`_. These can be loaded in Gilda via
+    :mod:`pyobo` like in the following example
+    (note, sometimes ITIS is unresponsive):
+
+    .. code-block:: python
+
+        import gilda, pyobo
```

Comment:
I would like to use obonet here since it's a built-in dependency. We can provide a PyOBO example in the notebook.